gwsurrogate/catalog.py

```diff
@@ -109,6 +109,17 @@
   '''https://journals.aps.org/prd/abstract/10.1103/PhysRevD.99.064045''',
   'b42cd577f497b1db3da14f1e4ee0ccd1')
 
+_surrogate_world['NRHybSur2dq15'] = \
+  surrogate_info(\
+  'https://zenodo.org/record/3348115/files/NRHybSur2dq15.h5',
+  '''Surrogate model for aligned-spin binary black holes with mass ratios q<=15
+  and spin magnitudes <=0.5. The model is trained on NR waveforms that have been
+  hybridized using EOB/PN and spans the entire LIGO frequency band. This model
+  is  presented in arxiv:2203.10109. Available modes are
+  [(2,2), (2,1), (3,3), (4,4), and (5,5)]. The m<0 modes are deduced from the m>0 modes.''',
+  '''FIXME URL''',
```

Comment:
What is this URL actually supposed to be?

Reply:
I think it's supposed to be the link to the paper. I was going to fix it later once it's accepted.